common/utils.js

```diff
@@ -43,8 +43,49 @@ var utils = {
     } else {
       return 'thru ' + thru;
     }
-  }
+  },
+
+  shuffle: function (arr) {
```

Comment:
No need to write your own shuffle function. Lodash is already included in this project and has its own: https://lodash.com/docs/2.4.2#shuffle

`_.shuffle`